js/app.js

```diff
@@ -363,16 +349,17 @@
 				});
 			}
 		},
+		addParticipantToCall: function(token, participant) {
+			return this.signaling.addParticipantToCall(token, participant);
```

Comment:
Well, that would require that all clients poll the (slow) PHP regularly about who is in the call and we want to avoid that by moving this to the standalone signaling server. The PHP backend is only used to check which rooms / calls exist.